airflow/api_connexion/endpoints/dag_endpoint.py

```diff
@@ -74,14 +74,15 @@ def get_dags(
     tags: Collection[str] | None = None,
     dag_id_pattern: str | None = None,
     only_active: bool = True,
+    only_unpaused: bool = False,
```

Comment:
This is a bit of a double negative here which we should avoid

Also it's not possible to list _only_ paused dags.

How about instead we have

`paused: bool | None = None` -- if it's None we don't filter, otherwise we filter by the given value?

Reply:
I agree, my first thought was to do exactly what you're suggesting, but I implemented `only_unpaused` to continue in the same pattern. I will update it